word_counter.rb

```diff
@@ -0,0 +1,79 @@
+require 'pragmatic_segmenter'
+# how do you include a gem such as this so that the program will run on a given computer?
+
+def word_counter
```

Comment:
- namespace this method within a class or module
- add a method comment
- this method is incorrectly named. Think of method names as verbs and nouns as classes. I haven't looked at the content of this method yet, but if it counts words, then call it `word_counter`. Still, I don't think that's a great name because I would assume a method called `word_counter` takes a string and returns an integer describing how many words are in the string. I assume this method does something much more complicated, so give it a name that reflects that complexity. Naming is important and goes a long way towards writing readable and maintainable code.